### PR TITLE
Add 1.3.0 release notes

### PR DESCRIPTION
**Pull request by philippjfr**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,56 @@
...
+- Guard undefined values from being set on `BrowserInfo` ([#5588](https://github.com/holoviz/panel/pull/5588))
+- Fix thumbnails and use Panel design on index page ([#5595](https://github.com/holoviz/panel/pull/5595))
+- Fix regressions in `TextEditor` caused by migration to shadow DOM ([#5609](https://github.com/holoviz/panel/pull/5609))
+- Sync `location` state from request ([#5581](https://github.com/holoviz/panel/pull/5581))
+- Fix `Select` widget label offset in Material Design ([#5639](https://github.com/holoviz/panel/pull/5639))
+- Override token contents when reusing sessions ([#5640](https://github.com/holoviz/panel/pull/5640))
+- Fix patching a table with a `DataFrame` with a custom index ([#5645](https://github.com/holoviz/panel/pull/5645))
+- Set `FloatPanel` status correctly on initialization ([#5651](https://github.com/holoviz/panel/pull/5651))
+- Fix patching table with `pd.Timestamp` values ([#5650](https://github.com/holoviz/panel/pull/5650))
+- Ensure `notifications` and `browser_info` are loaded when `HoloViews` is loaded
```
Comment (cdeil) [inline CHANGELOG.md:40]: Should this line link to #5657 ?

Also: version number in merge request: "1.13.0" -> "1.3.0"

Comment (philippjfr) [reply CHANGELOG.md:40]: Good catch, thanks!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,56 @@
...
+- Gracefully handle resolution of invalid paths in `_stylesheets` ([#5666](https://github.com/holoviz/panel/pull/5666))
+- Handle patching tables with `NaT` values ([#5675](https://github.com/holoviz/panel/pull/5675))
+
+### Compatibility
+
+- Upgrade to Param 2.0
```
Comment (cdeil) [inline CHANGELOG.md:46]: Suggest to write "Compatibility with Param 2.0" instead, which to me sounds clearer that older versions are still supported also.

Maybe mention that this release adds Python 3.12 support and drops Python 3.8 support as well?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,56 @@
...
+
+- Integrate support for param reactive expressions and expose `pn.rx` ([#5138](https://github.com/holoviz/panel/pull/5138), [#5582](https://github.com/holoviz/panel/pull/5582))
+- Implement `ChatMessage`, `ChatFeed` and `ChatInterface` components ([#5333](https://github.com/holoviz/panel/pull/5333))
+- Unify OAuth implementations and refresh `access_token` ([#5627](https://github.com/holoviz/panel/pull/5627))
+- Add `ColorMap` widget ([#5647](https://github.com/holoviz/panel/pull/5647))
+
+### Enhancement
+
+- Add unit to widget in `HoloViews` pane if provided ([#5535](https://github.com/holoviz/panel/pull/5535))
+- Allow registering global on_`session_destroyed` callback ([#5585](https://github.com/holoviz/panel/pull/5585))
```
Comment (cdeil) [inline CHANGELOG.md:19]: Typo, should be `on_sessno_destroyed`.

File: CHANGELOG.md (hunk 2 above)
Comment (maximlt) [reply CHANGELOG.md:46]: Hi @cdeil, Param 2.0 will be the minimum version required by Panel 1.3 so I think "Upgrade to Param 2.0" is correct:
https://github.com/holoviz/panel/blob/0067d401553b31d8a67646ad7946134ed8acce33/setup.py#L104

File: CHANGELOG.md (hunk 1 above)
Comment (philippjfr) [inline CHANGELOG.md:40]:
```suggestion
- Ensure `notifications` and `browser_info` are loaded when `HoloViews` is loaded ([#5657](https://github.com/holoviz/panel/pull/5657))
```

File: CHANGELOG.md (hunk 2 above)
Comment (philippjfr) [inline CHANGELOG.md:46]:
```suggestion
- Add support for Python 3.12 and drop Python 3.8 support
- Upgrade to Param 2.0 as minimum required version
```

**Pull request merged**